tests/tck/features/insert/InsertIfNotExists.feature

```diff
@@ -0,0 +1,170 @@
+# Copyright (c) 2021 vesoft inc. All rights reserved.
+#
+# This source code is licensed under Apache 2.0 License,
+# attached with Common Clause Condition 1.0, found in the LICENSES directory.
+Feature: Insert vertex and edge with if not exists
+
+  Scenario: insert vertex and edge if not exists test
+    Given an empty graph
+    And create a space with following options:
+      | partition_num  | 9                |
+      | replica_factor | 1                |
+      | vid_type       | FIXED_STRING(20) |
+    And having executed:
+      """
+      CREATE TAG IF NOT EXISTS person(name string, age int);
+      CREATE EDGE IF NOT EXISTS like(likeness int);
+      CREATE TAG IF NOT EXISTS student(grade string, number int);
+      """
+    # insert vertex succeeded
+    When try to execute query:
+      """
+      INSERT VERTEX person(name, age) VALUES "Tom":("Tom", 22)
+      """
+    Then the execution should be successful
+    When executing query:
+      """
+      INSERT VERTEX IF NOT EXISTS
+        person(name, age)
+      VALUES
+        "Conan":("Conan", 10),
+        "Yao":("Yao", 11),
+        "Conan":("Conan", 11);
+      """
+    Then the execution should be successful
+    # check vertex result with fetch
+    When executing query:
+      """
+      FETCH PROP ON person "Conan" YIELD person.age as age
+      """
+    Then the result should be, in any order, with relax comparison:
+      | VertexID | age |
+      | "Conan"  | 10  |
+    # insert vertex if not exists
+    When executing query:
+      """
+      INSERT VERTEX IF NOT EXISTS person(name, age) VALUES "Conan":("Conan", 20)
```

Comment:
After the execution is successful, is there any messege to indicate that the current data has not been successfully inserted？